intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/verifiers/PluginIdVerifier.kt

```diff
@@ -0,0 +1,57 @@
+package com.jetbrains.plugin.structure.intellij.verifiers
+
+import com.jetbrains.plugin.structure.base.problems.PropertyNotSpecified
+import com.jetbrains.plugin.structure.intellij.beans.PluginBean
+import com.jetbrains.plugin.structure.intellij.problems.IllegalPluginIdPrefix
+import com.jetbrains.plugin.structure.intellij.problems.PropertyWithDefaultValue
+import com.jetbrains.plugin.structure.intellij.problems.TemplateWordInPluginId
+
+val DEFAULT_ILLEGAL_PREFIXES = listOf("com.example", "net.example", "org.example", "edu.example", "com.intellij", "org.jetbrains")
```

Comment:
Maybe move to `companion object` of `PluginIdVerifier`?

Reply:
AFAIU `companion object`s that contain just the constants are now discouraged in favour of top-level constants.